packages/editor/src/components/editor/Editor.tsx

```diff
@@ -0,0 +1,39 @@
+import { Flex, ResizableHandle, ResizablePanel, ResizablePanelGroup, SidebarHeader, Toolbar, ToolbarTitle } from '@axonivy/ui-components';
+import { IvyIcons } from '@axonivy/ui-icons';
+import './Editor.css';
+
+export const Editor = () => {
+  // TODO: Variables coming from context
+  const masterTitle = 'Rest Clients Editor';
+  const masterContent = (
+    <Flex justifyContent='center' alignItems='center' style={{ height: '100%' }}>
+      <span>Content</span>
+    </Flex>
+  );
+  const detailTitle = 'Variables Editor - variableA';
+  const detailContent = (
+    <Flex justifyContent='center' alignItems='center' style={{ height: '100%' }}>
+      <span>Properties</span>
+    </Flex>
+  );
```

Comment:
better directly craft some components, then you can already use them in the jsx below.